examples/MNIST_classification.ipynb

```diff
@@ -0,0 +1,638 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Case study: MNIST hand-written digits dataset\n",
+    "\n",
+    "##### License: Apache 2.0\n",
+    "\n",
+    "\n",
+    "This notebook shows how to use topological data analysis to generate features for classifying digits."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Import libraries\n",
+    "The first step consists in importing relevant *gtda* components and other useful libraries or modules."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "_cell_guid": "b1076dfc-b9ad-4769-8c92-a6c4dae69d19",
+    "_uuid": "8f2839f25d086af736a60e9eeb907d3b93b6e0e5"
+   },
+   "outputs": [],
+   "source": [
+    "from gtda.images import Binarizer, Inverter, ImageToPointCloud, HeightFiltration, DilationFiltration, RadialFiltration, ErosionFiltration, SignedDistanceFiltration\n",
+    "from gtda.images import DensityFiltration\n",
```

Comment:
This line seems to be an error (the transformer isn't available in `giotto-tda`).